assets/css/application.scss

```diff
@@ -1035,4 +1035,10 @@ input.form-control.show-loader {
         border: 0;
     }
 
+    canvas,
+    .color-icon {
+        background: #fff;
+        filter: invert() hue-rotate(180deg);
```

Comment:
If I'm understanding it correctly, the [spec](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/invert()) requires that `invert()` be given an amount (`1` in this case). It might be coincidence that your browser assumed a value of `1`.